.eslintrc.json

```diff
@@ -1,3 +1,3 @@
 {
-    "extends": "next/core-web-vitals"
+    "extends": ["next/core-web-vitals", "plugin:react/jsx-runtime"]
```

Comment:
Was having some issues with eslint and parsing the `sx` prop with the Theme UI jsx, this seems to help.